src/OpenTelemetry.Abstractions/Trace/ITracer.cs

```diff
@@ -47,10 +49,184 @@ public interface ITracer
         IDisposable WithSpan(ISpan span);
 
         /// <summary>
-        /// Gets the span builder for the span with the given name.
+        /// Creates root span.
         /// </summary>
-        /// <param name="spanName">Span name.</param>
-        /// <returns>Span builder for the span with the given name.</returns>
-        ISpanBuilder SpanBuilder(string spanName);
+        /// <param name="operationName">Span name.</param>
+        /// <returns>Span instance.</returns>
+        ISpan StartRootSpan(string operationName);
```

Comment:
Can many of these methods on ITracer be turned into extension methods instead? Most of the implementations in Tracer.cs are redirections to other ITracer methods.

Reply:
this is great idea, I think it should be possible to keep all extensions in abstractions without the need to override them in SDK. I'll try to explore this path